xarray/core/dataset.py

```diff
@@ -5911,6 +5940,38 @@ def reduce(
         reduced : Dataset
             Dataset with this object's DataArrays replaced with new DataArrays
             of summarized data and the indicated dimension(s) removed.
+
+        Example
+        -------
+
+        import xarray as xr
+        import numpy as np
+
+        dataset = xr.Dataset(
+            {
+                'math_scores': (
+                    ['student', 'test'],
+                    [[90, 85, 92],
+                    [78, 80, 85],
+                    [95, 92, 98]]
+                ),
+                'english_scores': (
+                    ['student', 'test'],
+                    [[88, 90, 92],
+                    [75, 82, 79],
+                    [93, 96, 91]]
+                )
+            },
+            coords={
+                'student': ['Alice', 'Bob', 'Charlie'],
+                'test': ['Test 1', 'Test 2', 'Test 3']
+            }
+        )
+
+        mean_scores = dataset.reduce(func=np.mean, dim='test')
```

Comment:
This works, but would be a better demonstration of the usefulness of `.reduce` to reduce using a function that wasn't already available as a method (i.e. `Dataset.mean()` already exists).

Ideas:
- Some other numpy function
- Some scipy function (e.g. skewness)